src/website/app/demos/Icon/examples/rtl.js

```diff
@@ -22,7 +22,7 @@ export default {
   id: 'rtl',
   title: 'RTL',
   description:
-    'Some icons will be automatically mirrored for (right-to-left) RTL languages',
+    'Some icons will be [automatically mirrored](http://google.github.io/material-design-icons/#which-icons-should-be-mirrored-for-rtl) for RTL (right-to-left) languages depending on context.',
```

Comment:
Comma after "languages". Good idea to link that up.

Reply:
👍